### ISSUE TITLE
Fails to build on F-Droid

### ISSUE THREAD

**Issue by jspricke** [ubipo/osmfocus#8]
The latest version (1.0.3) fails to build:
```
* Where:
Build file '/home/vagrant/build/net.pfiers.osmfocus/build.gradle.kts' line: 40

* What went wrong:
java.lang.NullPointerException (no error message)
```
https://monitor.f-droid.org/builds/log/net.pfiers.osmfocus/103

could you change it that these variables are not needed?
 https://github.com/ubipo/osmfocus/blob/master/build.gradle.kts#L40

**Comment by ubipo** [comment]
Thanks for letting me know!

Fixed in 639f3d1.

Should I update something in fdroid/data or will the autoupdater still work?

**Comment by jspricke** [comment]
Thanks! It will be picked up automatically.
